src/main/java/io/jenkins/plugins/jfrog/JfStep.java

```diff
@@ -291,7 +292,10 @@ public String getDisplayName() {
 
         @Override
         public boolean isApplicable(Class<? extends AbstractProject> jobType) {
-            return true;
+            // JfStep Extends SimpleBuildStep, which applies also to FreeStyleProjects, as well as workflow jobs.
+            // To avoid errors due to unsupported inputs in FreeStyleProjects,
+            // Return applicable only for WorkflowJobProperty.
+            return WorkflowJobProperty.class.isAssignableFrom(jobType);
```

Comment:
Shouldn't be this?
```java
return WorkflowJob.class.isAssignableFrom(jobType) || WorkflowJob.class.isAssignableFrom(jobType);
```
I haven't tested the solution.
Please also verify with [Multibranch Pipelines](https://www.jenkins.io/doc/book/pipeline/multibranch/).

Reply:
It's strange, but when i debug i'm getting an error thrown, also on the main branch when it comes to multi-branch.
On the other hand, when i installed the plugin on my local Jenkins, it doesn't crash.
Maybe it's related to the Jenkins version.
It needs to be tested.